src/test/java/pages/BasePage.java

```diff
@@ -11,29 +11,28 @@
 
 public class BasePage {
 
-    WebDriver driver;
-    WebDriverWait wait;
-    Actions actions;
+    static WebDriver driver;
+    static WebDriverWait wait;
+    static Actions actions;
```

Comment:
I recommend: make the variables protected, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

```
    protected static WebDriver driver;
    protected static WebDriverWait wait;
    protected static Actions actions;
```